source/main/modules/window-manager/index.ts

```diff
@@ -207,18 +213,50 @@ export default class WindowManager {
   /**
    * Shuts down the window manager and performs final operations
    */
-  shutdown (): void {
+  shutdown(): void {
     this._persistWindowPositions()
   }
 
   /**
    * Listens to events on the main window
    */
-  private _hookMainWindow (): void {
+  private _hookMainWindow(): void {
     if (this._mainWindow === null) {
       return
     }
 
+    this._mainWindow.on('show', async () => {
+      // issue : Add tray to MacOS #4
+      // If Zettlr is running AND the Show app in the notification area is true, 
+      // the tray icon will be visible (regardless if Zettlr window(s) are visible or not).
+      // Zettlr can take a while a load. Only show the tray when the Zettlr window is ready.
+      // Do not add the actions (event handlers) to tray click and tray menu. See separate issue.
```

Comment:
Remove `issue : Add tray to MacOS #4` because #4 is reference to our team's fork only.
Remove `// Do not add the actions (event handlers) to tray click and tray menu. See separate issue.` because this is our teams communication.

Reply:
This will remove in new commit